Resources/Prototypes/_LostParadise/Entities/Clothing/OuterClothing/armor.yml

```
@@ -55,3 +55,55 @@
     reflectProb: 0.9
     reflects:
       - Energy
+
+- type: entity
+  parent: [ClothingOuterBaseLarge, AllowSuitStorageClothing, ClothingOuterEVASuitBase]
+  id: LPPlothingOuterArmorSwatAlternative
+  name: Lightened swat suit
+  description: Composed of semi-flexible polycarbonate, reinforced materials, and integrated ballistic plating, it offers exceptional protection against melee, bullet, and environmental threats, ensuring officers remain combat-ready in the harshest conditions.
+  components:
+  - type: Sprite
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)